nengo_loihi/builder/connection.py

```diff
@@ -212,7 +218,7 @@ def build_host_to_chip(model, conn):
     logger.debug("Creating DecodeNeuron ensemble for %s", conn)
     ens = model.node_neurons.get_ensemble(dim, add_to_container=False)
     ens.label = None if conn.label is None else "%s_ens" % conn.label
-    _inherit_seed(host, ens, model, conn)
+    _set_seed(host, ens, seed=rng, seeded=model.seeded[conn])
```

Comment:
Previously, we never needed a seed for this ensemble, because everything was fixed (encoders, gains, biases). So I think this was setting the seed to be `None` for the ensemble, which was fine.

Now, we use randomness (potentially) when generating neuron initial values. Furthermore, the seed we use for that `rng` is the ensemble seed plus one, so if the ensemble seed is `None` that's a problem.

I don't think it actually matters what the seed is, because I've set DecodeNeurons to have zero initial voltages elsewhere in this PR. But if that ever changes, I think this should work fine. (This ensemble doesn't exist if you're just running in Nengo, since there are no DecodeNeurons there, so there's no worry about it being initialized the same as in Nengo.)

Reply:
But we're using `rng = np.random.RandomState(model.seeds[conn])` above. So if `model.seeds[conn]` is None then that isn't seeding anything either, and if it's not None then it should have been seeded correctly before this change shouldn't it?

Reply:
It's not about it being seeded; you're right that this still allows `None` and this to not be seeded (which I think is fine). It's that the Nengo builder does something like `ensemble_seed + 1` to get the seed for that ensemble's neurons, so if the seed is `None` it actually breaks. This avoids having the seed be `None`.

Reply:
Pushed a fixup to make that a bit more explicit (returns things to mostly the way they were before, but makes sure the seed is not None). We could optionally only do that if `dest_obj` is an Ensemble, but I don't think there's any downside in doing it in all cases so I just did that.